Cargo.toml

```diff
@@ -54,7 +55,7 @@ exclude = [
 ]
 
 # The `default-members` are the things that are built/tested when you run `cargo
-# build`, `cargo test` (or most of the cargo commands not provided by a
+# build`, `cargo test` (or most of the cargo commands not provided by ca
```

Comment:
What happened here?

Reply:
Probably had the wrong pane of VSC focussed without realizing it.